src/library.rs

```diff
@@ -192,20 +215,25 @@ pub struct Alias {
     pub c_identifier: String,
     pub typ: TypeId,
     pub target_c_type: String,
+    pub doc: Option<String>,
```

Comment:
Would you mind doing the same with `doc-deprecated`? It is relevant for doc generation and parser-wise is exactly like `doc`.